### DISCUSSION TITLE
Any plans for Relationships?

### DISCUSSION THREAD

**Discussion by slimshader** [Doraku/DefaultEcs#174] [Ideas]
Hi, are there a plans to support ecs Relationships as described here : 

https://ajmmertens.medium.com/building-games-in-ecs-with-entity-relationships-657275ba2c6c

https://flecs.docsforge.com/latest/relationships-manual/

**Comment by Doraku** [comment]
I do have this feature in mind:

> add a special component type to reference an other entity component

which could probably be used afterward to build a relationship api but no estimate for when it will come sorry :/
